### PR TITLE
list assigned roles

### PR DESCRIPTION
**Pull request by zGrav**
Marking this for issue #26

### CODE REVIEW

**Comment by zGrav** [comment]
Marking this for issue #26

**Comment by zGrav** [comment]
Latest commit with ID 3b412d5 is due to there already being a who has "role" role? command, so I decided to keep it simple.

Anywhom, the test script doesn't seem to be liking something regarding roles, apparently, it's not setting them properly?

@ceci-woodward mind taking a look when possible please?

File: src/auth.coffee [hunk 1]
```diff
@@ -64,6 +65,20 @@ module.exports = (robot) ->
 
   robot.auth = new Auth
 
+  robot.respond /list assigned roles/i, (msg) ->
```
Comment (nick-woodward) [inline src/auth.coffee]: Move this command to the bottom of the file.

For some reason if this command isn't after the command that adds roles to users then the roles object isn't populated.

Comment (zGrav) [inline src/auth.coffee]: Interesting.

Fixed, pushing now.

Comment (nick-woodward) [inline src/auth.coffee]: It doesn't really make any sense, but for some reason that's how its working.

**Comment by nick-woodward** [comment]
Looks good to me however I don't have the ability to merge PRs.

@patcon
Anything you'd like to add?

**Comment by patcon** [comment]
If you guys are in agreement, happy to merge! Thanks gents!

**Comment by zGrav** [comment]
All fine with me.

Appreciate a lot the help and collaboration @ceci-woodward and @patcon !

**Pull request merged**